src/pages/Commits.tsx

```diff
@@ -28,10 +28,10 @@ import {
 } from "types/commits";
 import { TaskStatus } from "types/task";
 import { array, queryString } from "utils";
+import { ProjectSelect } from "../components/projectSelect";
```

Comment:
you should use an absolute import `components/projectSelect`

Reply:
done